WcaOnRails/app/views/registrations/index.html.erb

```diff
@@ -1,77 +1,79 @@
 <% provide(:title, "Registrations for #{@competition.name}") %>
 
 <%= render layout: "nav" do %>
-  <%= wca_table data: { toggle: "table", sort_name: "name" } do %>
-    <thead>
-      <tr>
-        <th class="name" data-sortable="true" data-field="name">Name</th>
-        <th class="wca-id" data-sortable="true">WCA ID</th>
-        <th class="country" data-sortable="true">Citizen of</th>
+  <% cache ["registrations_index", @registrations.pluck(:id), @registrations.maximum(:updated_at)] do %>
```

Comment:
From what I could see in the controller, it's just whatever order the DB chooses for `@competition.registrations`, and I think it gets sorted by JS on the page.

So if the ids are [1,2,3] once and [3,1,2] the next time, we're needlessly re-rendering.

In practice, MySQL is probably always returning in id order, and it ends up fine. Postgres is far less predictable.